apps/frontend/src/pages/chat/libs/components/buttons-controller/libs/components/suggestions-manipulation-options/suggestions-manipulation-options.tsx

```diff
@@ -0,0 +1,65 @@
+import { Button, Icon } from "~/libs/components/components.js";
+import {
+	useAppDispatch,
+	useAppSelector,
+	useCallback,
+} from "~/libs/hooks/hooks.js";
+import { actions as chatActions } from "~/modules/chat/chat.js";
+import { buttonsModeOption } from "~/pages/chat/libs/enums/enums.js";
+
+import {
+	suggestionsManipulationButtonLabel,
+	suggestionsManipulationMessages,
+} from "./libs/enums/enums.js";
+import styles from "./styles.module.css";
+
+const SuggestionsManipulationOptions: React.FC = () => {
+	const { taskSuggestions, threadId } = useAppSelector((state) => state.chat);
```

Comment:
```suggestion
	const { taskSuggestions, threadId } = useAppSelector((state) => ({
	  taskSuggestions: state.chat.taskSuggestions,
	  threadId: state.chat.threadId
	});
```
Only return what you need